bentoml/_internal/io_descriptors/numpy.py

```diff
@@ -251,6 +251,254 @@ async def to_http_response(self, obj: ext.NpNDArray, ctx: Context | None = None)
         else:
             return Response(json.dumps(obj.tolist()), media_type=MIME_TYPE_JSON)
 
+    def handle_tuple(self, proto_tuple):
+        """
+        Convert given protobuf tuple to a tuple list
+        """
+        from google.protobuf.duration_pb2 import Duration
+        from google.protobuf.timestamp_pb2 import Timestamp
+
+        from ..generated_pb import io_descriptors_pb2
```

Comment:
Actually for easier naming and imports, lets just make this to `bentoml.protos`? @Proto007 It is best to follow practice and not be inventive for naming convention.

Reply:
Are you suggesting the change from `generated_pb` to `protos`? So it will be something like `from bentoml.protos import io_descriptors_pb2`?

Reply:
Or maybe `from ..protos import io_descriptors_pb2`?

Reply:
for protos we can do absolute imports. It is more clear that way.

Reply:
Can you clarify the location of the generated `_pb2.py` files?

Reply:
Done.